addons/hr/models/res_partner.py

```diff
@@ -9,6 +10,7 @@ class ResPartner(models.Model):
         'hr.employee', 'work_contact_id', string='Employees', groups="hr.group_hr_user",
         help="Related employees based on their private address")
     employees_count = fields.Integer(compute='_compute_employees_count', groups="hr.group_hr_user")
+    employee = fields.Boolean(help="Whether this contact is an Employee.", compute='_compute_employee', store=True)
```

Comment:
Don't store the field, there is too many res.partner on a production db (like odoo.com), a search method is sufficient.

Reply:
Hello, the field is stored in the base module. Should I make it non-stored in hr? Also making it non-stored will cause read_group to not work properly, so I think it is better to not change it. What do you think?